src/main/java/com/mathworks/ci/RunMatlabTestsStep.java

```diff
@@ -121,7 +121,9 @@ public String getDisplayName() {
     
     private String getInputArgs() {
         final List<String> inputArgs = new ArrayList<>();
-        final Map<String, String> args = getMatlabArgs();
+        final Map<String, String> args = getGenscriptArgs();
+        
+        inputArgs.add("'Test'");
```

Comment:
You have a check for `inputArgs.isEmpty()` below here that seems like it would never occur given this line.

Reply:
Doe.